sbndqm/dqmAnalysis/CRT/SBND/BernCRTdqmSBND_module.cc

```diff
@@ -339,182 +327,365 @@ if(!fragmentHandle.isValid() || fragmentHandle->size() == 0)
     ///////////////////////////
    
     auto currflag = hit.flags;
-    int maxindex = -1;
     for(int i = 0; i<32; i++) {
       if (currflag == 3) {
-        sbndaq::BernCRTdqmSBND::flag3channel[i]++;
         flag3hit++;
       }
-      totaladc  += adc[i];
+      //totaladc  += adc[i]; // We'll account for totaladc later
       ADCchannel = adc[i];
-      sbndaq::BernCRTdqmSBND::pedSum[i] += adc[i];
-      if (adc[i] > pedMax[i]) {pedMax[i] = adc[i];}
-      if (adc[i] > ped2Max[i]) {
-        if (adc[i] < pedMax[i]) {
-          ped2Max[i] += adc[i];
-        }
-      }
-      sbndaq::BernCRTdqmSBND::pedSumSq[i] += adc[i]*adc[i];
-      sbndaq::BernCRTdqmSBND::pedNHits[i]++;
       uint64_t lastbighitchannel = fragment_timestamp -sbndaq::BernCRTdqmSBND::lastbighit[i];
```

Comment:
I still don't think these variables make sense... We're not specifying the board here so we're just grabbing any random channel i from any board.

This is why I removed a lot of this...

Reply:
Hmm. Could go two ways - either we add an external index [mac5] and populate/send lastbighit[mac5][i] to the appropriate index 100*mac5 + i, or we get rid of this

Reply:
Nuking this for now but might want to revisit maybe?

Reply:
Agreed!